modules/builds-source-secret-trusted-ca.adoc

```diff
@@ -5,15 +5,15 @@
 [id="builds-source-secret-trusted-ca_{context}"]
 = Creating a secret from source code trusted certificate authorities
 
-The set of TLS certificate authorities (CA) that are trusted during a `git clone` operation are built into the {product-title} infrastructure images. If your Git server uses a self-signed certificate or one signed by an authority not trusted by the image, you can create a secret that contains the certificate or disable TLS verification.
+The set of Transport Layer Security (TLS) certificate authorities (CA) that are trusted during a Git clone operation are built into the {product-title} infrastructure images. If your Git server uses a self-signed certificate or one signed by an authority not trusted by the image, you can create a secret that contains the certificate or disable TLS verification.
```

Comment:
BTW, thanks for spelling out the first instance of acronyms throughout these docs!!